tamr_client/schema_mapping/project.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional
+
+from tamr_client import project
+from tamr_client._types import (
+    Instance,
+    JsonDict,
+    Project,
+    SchemaMappingProject,
+    Session,
+    URL,
+)
+
+
+def _from_json(url: URL, data: JsonDict) -> SchemaMappingProject:
+    """Make schema mapping project from JSON data (deserialize)
+
+    Args:
+        url: Project URL
+        data: Project JSON data from Tamr server
+    """
+    return SchemaMappingProject(
+        url, name=data["name"], description=data.get("description")
+    )
+
+
+def create(
+    session: Session,
+    instance: Instance,
+    name: str,
+    description: Optional[str] = None,
+    external_id: Optional[str] = None,
+    unified_dataset_name: Optional[str] = None,
+) -> Project:
+    """Create a Schema Mapping project in Tamr.
+    Args:
```

Comment:
Needs whitespace for doc formatting:

```suggestion

    Args:
```

Reply:
updated.